models/classes/preview/TestPreviewerService.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoTests\models\preview;
+
+use oat\oatbox\AbstractRegistry;
+use oat\tao\model\modules\DynamicModule;
+use oat\tao\model\ClientLibConfigRegistry;
+use oat\oatbox\service\ConfigurableService;
+
+/**
+ * Class TestPreviewerService
+ *
+ * @package oat\taoTests\models\preview
+ */
+class TestPreviewerService extends ConfigurableService
```

Comment:
as we discussed there are two exactly the same services `ItemPreviewerService` and the new one `TestPreviewerService`
it really makes sense to merge them.
I think you can create a common class under `tao`
ClientLibConfigRegistryManager which can take REGISTRY_ENTRY_KEY and PREVIEWERS_KEY  as construct parameters
and maybe make a factory for it